src/style.css

```diff
@@ -0,0 +1,124 @@
+html {
+  --form-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  --form-shadow-hover: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  src: url("fonts/Avenir-Book.ttf") format("truetype");
+  font-family: "Avenir";
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  src: url("fonts/Avenir-Heavy.ttf") format("truetype");
+  font-family: "Avenir";
+  font-style: normal;
+  font-weight: 600;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+}
+
+.search-form {
+  display: flex;
+  justify-content: center;
+  margin: 20px auto 0;
+}
+
+.search-form_big {
+  width: 100%;
+  height: 70px;
+}
+
+.search-form_small {
+  width: 100%;
+  height: 42px;
+}
+
+.search-form__label {
+  position: relative;
+  display: block;
+  width: 100%;
+}
+
+.search-form__label:after {
+  position: absolute;
+  content: "";
+  display: block;
+}
+
+.search-form__label_big:after {
+  background-image: url("images/Search.svg");
+  background-repeat: no-repeat;
+  background-size: contain;
```

Comment:
the same here, you can move these styles to `.search-form__label:after`

Reply:
+